packages/components/inputs/selectable-search-input/src/selectable-search-input.tsx

```diff
@@ -221,6 +221,12 @@ export type TSelectableSearchInputProps = {
    * eg: inputDataProps={[{ 'prop-1': 'value-1' }, { 'prop-2': 'value-2' }]}
    */
   inputDataProps?: Record<string, string>;
+  /**
```

Comment:
It would be great to add a new test verifying the provided custom components are actually used?

Reply:
updated hereL https://github.com/commercetools/ui-kit/pull/2831/commits/aa3cd25ca24a4ba0a42c335678e0d72d346457a3

Thank you